tests/parm/ww3_shel.nml.IN

```diff
@@ -0,0 +1,20 @@
+&input_nml
```

Comment:
This PR does not require that the nml files be used. An nml file will be used only if present. The PR to update the meshcap (#1366) passes all tests, even though it is capable of using the nml file, because no inp file is present for now. Similarly, the current workflows and regression tests can continue to use the inp files.

Reply:
So I understood incorrectly that the ufs-weather-model regression tests will not be moving away from the ww3_shel.inp files then?

Reply:
The RTs for the weather-model will be using nml files after this PR. But any other workflow can continue to use the inp file.  I don't understand how this feature implemented in UFS-RTs breaks any other use-case.

Reply:
If we are using nml instead of inp in the ufs-weather-model, we cannot be sure that the workflow which is tracking the ufs-weather-model is in sync.  We can no longer assume that what's tested in the ufs-weather-model is valid on the workflow side.  This will also make it very difficult for non-wave developers to track changes and know how to update the workflow for ufs-weather-model updates.   Moreover, to ensure we're testing things properly on the WW3 standalone side, this will force the WW3 standalone tests to move to also have to test nml files. This will double the amount of testing and work required for the WW3 code managers who are already stretched very thin.  @MatthewMasarik-NOAA and I discussed at lengths strategies around having some programs (or situations) using nml while others use inp files and could not figure out a way to do that without significantly increasing our workload for code management beyond what we have capacity for at this time.   This is why I'm requesting that we continue with the inp files in the ufs-weather-model for now until we can move all WW3 programs, WW3 standalone testing and  workflows to use nml files.

Reply:
I don't understand what this change has to do w/ the standalone testing. This change applies only to the optional use of nml files for the mesh cap, which is used only for the three applications tested here in the UWM (global, hafs, the new cmeps-atmwav).

Reply:
If ufs-weather-model is using nml and not inp files, WW3 standalone tests also need to be testing nml in addition to inp files in the WW3 standalone testing (thus doubling time and effort). In addition, this will also force the global-workflow into using nml files as it needs to stay in sync with the ufs-weather-model.  This is also additional unexpected work that will take time.  Unfortunately we're just not set-up for a mixture of nml and inp files right now.   If we can revert line 273 in tests/run_test.sh then I'm okay with this PR.

Reply:
Question: Does the code need to use .nml or .inp (i.e. the change will force the use of .nml)? or will it default to one (say .nml) and if it doesn't see it, then go to .inp?

Reply:
@BrianCurtis-NOAA If there is no nml file, it will use the inp file. One or the other (inp or nml) has to be present.